cli/azd/pkg/infra/provisioning/bicep/bicep_provider.go

```diff
@@ -882,6 +913,25 @@ func (p *BicepProvider) Destroy(ctx context.Context, options DestroyOptions) (*D
 		)
 	}
 
+	var emptyTemplate json.RawMessage
+	if targetScope == azure.DeploymentScopeSubscription {
+		emptyTemplate = []byte(cEmptySubDeployTemplate)
+	} else {
+		emptyTemplate = []byte(cEmptyResourceGroupDeployTemplate)
+	}
+
+	// create empty deployment to void provision state
+	// We want to keep the deployment history, that's why it's not just deleted
+	if _, err := p.deployModule(ctx,
+		deployScope,
+		emptyTemplate,
+		azure.ArmParameters{},
+		map[string]*string{
+			azure.TagKeyAzdEnvName: to.Ptr(p.env.GetEnvName()),
```

Comment:
Perhaps we should add a tag here that indicates this deployment was done because of an `azd down` operation, so we can distinguish from someone who did an empty provision?  We might also want to update some of our existing logic that walks the deployment history to be aware of these tombstones and do something sensible (like abort in the same way as if a deployment could not be found)?

Reply:
done